debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -50,7 +52,7 @@
 
 /**
  * A component that reads the binlog of a MySQL server, and records any schema changes in {@link MySqlSchema}.
- * 
+ *
```

Comment:
Can you remove unrelated changes like this from your commits? Ideally, the pull request only changes the minimum number of lines. Let me know if you need an easy way to do this.